Source/Python/WarpXWrappers.cpp

```diff
@@ -130,6 +130,13 @@ extern "C"
     {
         warpx_amrex_init(argc, argv, true, mpicomm);
     }
+#else
```

Comment:
I think this can be simplified by removing the `#ifdef BL_USE_MPI` macro (and the `else` block as well) so that the routine is always compiled. As you say, this routine shouldn't be called in serial but it needs to be declared, so a special version of it is not needed for serial.